tmail-backend/tmail-third-party/open-ai/AIBotMailet.adoc

```diff
@@ -0,0 +1,24 @@
+= AIBotMailet
+
+`AIBotMailet` is a mailet designed to automatically reply to emails sent to a specific recipient (e.g., gpt@linagora.com). It uses the OpenAI API by default but can be configured to use any LLM API supported by https://github.com/langchain4j/[langchain4j].
+
+== Configuration Example
+
+[source,xml]
+----
+<mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@tmail.com" class="com.linagora.tmail.mailet.AIBotMailet">
+      <apiKey>{put you api key here}</apiKey>
+      <gptAddress>gpt@tmail.com</gptAddress>
+      <model>linagoras-chitchat</model>
+      <baseURL>https://ai.linagora.com/api</baseURL>
+</mailet>
+----
+
+=== Parameters
+
+The mailet configuration requires the following parameters:
+
+* `apiKey`: The API key for accessing the LLM API service.
+* `gptAddress`: The email address used to send replies generated by the LLM model.
```

Comment:
Now that I think about it again (sorry to be picky^^): as we are now making a more generic API that could use any LLM languages and not just chatGPT, does that parameter name make sense?

Maybe something like `botAddress` would make more sense for example?

Reply:
You are right, the parameter doesn't make sense anymore, it should be good now.